### DISCUSSION TITLE
Is it possible to transfer kernels and estimators across datasets?

### DISCUSSION THREAD

**Discussion by dawe** [theislab/cellrank#774] [General]
Hi all, I am analyzing a scRNA-seq dataset with `cellrank`. Results are good or, at least, interesting.
Now, for the same cells I could generate a count matrix on totally different features and I would like to, at least, retrieve the lineage drivers from this new features using the estimates made on RNA. I can transfer from one `AnnData` to the other all the informations stored in `.obs`, `.obsm` and possibly `.uns`, but it seems that I have to recompute a new kernel and a new estimator.
I have tried to substitute the `adata` reference in the estimator (`g_fwd.adata = new_adata`) but `compute_lineage_drivers()` complains about different sizes in features (which is kind of expected...).
I am now looking at the source code, I may try to call `cellrank` functions directly, but any suggestion is very welcome.
Anyhow, what would be the best way to transfer kernels and estimators on different views of the same data? This may be relevant when multimodal data are available (not my case, though).

**Comment by dawe** [comment]
I've found that calling `cr.tl._utils._correlation_test()` directly works for me, given that `data` is the new `AnnData` and `lin_probs` is the estimator `.absorption_probabilities`.

**Reply by Marius1311** [reply]
Nice, does that solve your issue?

**Reply by dawe** [reply]
It does. Now I only have to figure out if results make any sense, but that is a completely different story 😄
